C/services/core/include/configuration_manager.h

```diff
@@ -26,25 +26,25 @@ class ConfigurationManager {
 		ConfigCategory			createCategory(const std::string& categoryName,
 							       const std::string& categoryDescription,
 							       const std::string& categoryItems) const;
-		// Called by GET /foglamp/category/{categoryName}/{configItem}
+		// Called by GET /foglamp/service/category/{categoryName}/{configItem}
```

Comment:
Not quite true, it can be called either via the microservice management API or the admin API

Reply:
Fixed in 2f7b2f07a3a0308dbee52a1781360f5455b32e92